jbpm-console-ng-dashboard/jbpm-console-ng-dashboard-client/src/main/java/org/jbpm/dashboard/renderer/client/panel/ProcessDashboardViewImpl.ui.xml

```diff
@@ -38,20 +38,21 @@
     }
 
     .header {
-      padding-left: 10px;
-      padding-bottom: 20px;
+      padding: 15px;
       background-color: white;
     }
 
+    .breadcrumb {
```

Comment:
Good point. I'm going to see if the standard pfly breadcrumb cab be used. Thanks.